docs/release_notes/ibexa_dxp_v5.0_deprecations.md

```diff
@@ -0,0 +1,572 @@
+| `ezuser`                                                | `ibexa_user`                                                              |
+| `ezuser_accountkey`                                     | `ibexa_user_accountkey`                                                   |
+| `ezuser_role`                                           | `ibexa_user_role`                                                         |
+| `ezuser_setting`                                        | `ibexa_user_setting`                                                      |
+
+## Field type identifiers
+
+Several field type identifiers have changed.
+
+| Old identifier (`legacy_alias`) | New identifier (`alias`)        |
+|:--------------------------------|:--------------------------------|
+| `ezauthor`                        | `ibexa_author`                    |
+| `ezbinaryfile`                    | `ibexa_binaryfile`                |
+| `ezboolean`                       | `ibexa_boolean`                   |
+| `ezcontentquery`                  | `ibexa_content_query`             |
+| `ezcountry`                       | `ibexa_country`                   |
+| `ezdate`                          | `ibexa_date`                      |
+| `ezdatetime`                      | `ibexa_datetime`                  |
+| `ezemail`                         | `ibexa_email`                     |
+| `ezfloat`                         | `ibexa_float`                     |
+| `ezform`                          | `ibexa_form`                      |
+| `ezgmaplocation`                  | `ibexa_gmap_location`             |
+| `ezimage`                         | `ibexa_image`                     |
+| `ezimageasset`                    | `ibexa_image_asset`               |
+| `ezinteger`                       | `ibexa_integer`                   |
+| `ezisbn`                          | `ibexa_isbn`                      |
+| `ezkeyword`                       | `ibexa_keyword`                   |
+| `ezlandingpage`                   | `ibexa_landing_page`              |
+| `ezmatrix`                        | `ibexa_matrix`                    |
+| `ezmedia`                         | `ibexa_media`                     |
+| `ezobjectrelation`                | `ibexa_object_relation`           |
+| `ezobjectrelationlist`            | `ibexa_object_relation_list`      |
+| `ezrichtext`                      | `ibexa_richtext`                  |
+| `ezselection`                     | `ibexa_selection`                 |
+| `ezstring`                        | `ibexa_string`                    |
+| `eztext`                          | `ibexa_text`                      |
+| `eztime`                          | `ibexa_time`                      |
+| `ezurl`                           | `ibexa_url`                       |
+| `ezuser`                          | `ibexa_user`                      |
+
+## PHP API classes and methods
+
+### `ibexa/admin-ui`
```

Comment:
Let's add an entry for `ibexa/user`, mentioning https://github.com/ibexa/user/pull/99